highcharts/src/main/scala/react/highcharts/Chart.scala

```diff
@@ -23,7 +23,7 @@ import scala.scalajs.js.|
 trait WrapProceed extends js.ThisFunction0[Chart_, Unit]
 
 @js.native
-@JSImport("highcharts/es-modules/masters/highcharts.src.js", JSImport.Default)
+@JSImport("highcharts", JSImport.Default)
```

Comment:
Admittedly, not sure about this. But if it causes problems downstream we should try and fix it there I think, because this import seems more generic/correct. Idk. hacks on hacks ...